tutorials/integrations/bedrock_tracing_tutorial.ipynb

```diff
@@ -0,0 +1,277 @@
+    "\n",
+    "Calling `invoke_model` using the `uninstrumented_client` will produce no traces, but will show the output from the LLM."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      " I'm doing well, thanks for asking!\n"
+     ]
+    }
+   ],
+   "source": [
+    "prompt = b'{\"prompt\": \"Human: Hello there, how are you? Assistant:\", \"max_tokens_to_sample\": 1024}'\n",
+    "response = uninstrumented_client.invoke_model(modelId=\"anthropic.claude-v2\", body=prompt)\n",
+    "response_body = json.loads(response.get(\"body\").read())\n",
+    "print(response_body[\"completion\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "LLM calls using the `instrumented_client` will print traces to the console! By configuring the `SpanProcessor` to export to a different OpenTelemetry collector, your OpenInference spans can be collected and analyzed to better understand the behavior of your LLM application."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\n",
+      "    \"name\": \"bedrock.invoke_model\",\n",
+      "    \"context\": {\n",
+      "        \"trace_id\": \"0x99786bf598cbe43c575e3552cb432257\",\n",
+      "        \"span_id\": \"0x24e9e30ec5205b1a\",\n",
+      "        \"trace_state\": \"[]\"\n",
+      "    },\n",
+      "    \"kind\": \"SpanKind.INTERNAL\",\n",
+      "    \"parent_id\": null,\n",
+      "    \"start_time\": \"2024-02-14T21:47:31.552335Z\",\n",
+      "    \"end_time\": \"2024-02-14T21:47:33.147729Z\",\n",
+      "    \"status\": {\n",
+      "        \"status_code\": \"UNSET\"\n",
+      "    },\n",
+      "    \"attributes\": {\n",
+      "        \"llm.prompts\": \"Human: Hello there, how are you? Assistant:\",\n",
+      "        \"llm.invocation_parameters\": \"{\\\"max_tokens_to_sample\\\": 1024}\",\n",
+      "        \"llm.model_name\": \"anthropic.claude-v2\",\n",
+      "        \"message.content\": \" I'm doing well, thanks for asking!\"\n",
+      "    },\n",
+      "    \"events\": [],\n",
+      "    \"links\": [],\n",
+      "    \"resource\": {\n",
+      "        \"attributes\": {},\n",
+      "        \"schema_url\": \"\"\n",
+      "    }\n",
+      "}\n",
+      " I'm doing well, thanks for asking!\n"
+     ]
+    }
+   ],
+   "source": [
+    "response = instrumented_client.invoke_model(modelId=\"anthropic.claude-v2\", body=prompt)\n",
+    "response_body = json.loads(response.get(\"body\").read())\n",
+    "print(response_body[\"completion\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The above trace has also been exported to Phoenix where it can be viewed http://localhost:6006/tracing"
```

Comment:
```suggestion
    "The above trace has also been exported to Phoenix where it can be viewed http://localhost:6006/tracing"
```
hardcoding localhost is probably not a good idea as that will be wrong in colab.  you can get the base url using px.active_session().url